adapter-base/pom.xml

```diff
@@ -41,19 +45,31 @@
     </dependency>
     <dependency>
       <groupId>org.eclipse.hono</groupId>
-      <artifactId>hono-client-command</artifactId>
+      <artifactId>hono-client-command-amqp</artifactId>
     </dependency>
     <dependency>
       <groupId>org.eclipse.hono</groupId>
-      <artifactId>hono-client-registry</artifactId>
+      <artifactId>hono-client-registry-amqp</artifactId>
     </dependency>
     <dependency>
       <groupId>org.eclipse.hono</groupId>
-      <artifactId>hono-client-telemetry</artifactId>
+      <artifactId>hono-client-telemetry-amqp</artifactId>
     </dependency>
     <dependency>
       <groupId>org.eclipse.hono</groupId>
-      <artifactId>hono-service-base</artifactId>
+      <artifactId>hono-client-command-kafka</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-client-notification-amqp</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-client-notification-kafka</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-client-telemetry-kafka</artifactId>
```

Comment:
can we order these dependencies so that it is easier to see if both variants (amqp and kafka) are being used?